CactbotOverlay/CactbotOverlay.csproj

```diff
@@ -66,6 +66,10 @@
     <Reference Include="OverlayPlugin.Core">
       <HintPath>ThirdParty\OverlayPlugin\OverlayPlugin.Core.dll</HintPath>
     </Reference>
+    <Reference Include="OverlayPlugin.Updater, Version=0.11.5.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
[v0.11.6](https://github.com/ngld/OverlayPlugin/releases/tag/v0.11.6) was released while this has been awaiting review.

Reply:
I'm still waiting on https://github.com/ngld/OverlayPlugin/tree/feature/newUpdater to get merged.